### ISSUE TITLE
RPC documentation

### ISSUE THREAD

**Issue by PavelNiedoba** [Bitcoin-ABC/bitcoin-abc#263]
Since bitcoincore RPC is changing quite a lot in 0.17 and is documented nicely here https://bitcoincore.org/en/doc/ I would expect that bitcoin-abc is porting this changes and follows same numbering, but by numbers now bitcoin-abc is ahead bitcoincore yet not implementing `getaddressesbylabel` which should be there. 

Therefore we need up to date RPC documentation. I suppose RPC.md file in this repo/doc is good place for it.

**Comment by PavelNiedoba** [comment]
Method 'getaddressesbyaccount' is deprecated, but I don't see working substitute for it right now. Solution from bitcoind 'getaddressesbylabel' is not working in 18.6, obviously not implemented.  I red https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/doc/release-notes.md, tried 'getlabeladdress', but it's not what I want. 

I would appreciate some help from bitcoin-abc development team, since we need this at simplecoin.eu to maintain exchange service for BCH and provide liquidity, thus increase BCH price.

**Comment by markblundeberg** [comment]
Yes I was just starting to develop a system and noticed that accounts are deprecated, without any replacement. Fortunately I did find a workaround that doesn't require any address labelling system at all.

I believe ABC team is right now trying to backport a lot of things from core, so one day they might catch up.

For now the only way to see RPC documentation appears to be running `bitcoin-cli help` and `bitcoin-cli help <cmd>`.

**Comment by microftech65** [comment]
https://github.com/ChristopherA/Learning-Bitcoin-from-the-Command-Line/blob/master/06_4_Sending_a_Transaction_with_a_Locktime.md
